packages/web-frontend/cypress/scripts/generateConfig/index.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
I this required to call this script from `yarn`?

Reply:
I believe it is, since the shebang isn't there, it will run it as a bash script by default

Reply:
Actually, they are not required since we invoke those with `babel-node` or (`node`)... This is a problem only if the package is a field itself: see https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bin

I recently encountered this issue in a personal project, that's why I updated those... I believe it's still good practice to include the shebang to indicate the script runner, what do you think?